pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -124,6 +127,9 @@ type manager struct {
 
 	// stateFileDirectory holds the directory where the state file for checkpoints is held.
 	stateFileDirectory string
+
+	// assignableCPUs is the set of online CPUs as reported by the system
+	assignableCPUs cpuset.CPUSet
```

Comment:
Is there a reason not to call this variable `allocatableCPUs` to make it match the API call now that we've settled on the name `Allocatable` for the API call?

Reply:
The only reason is to minimize the change internal to the module, which is a debatable reason (see below) and which I'm fine to change.